erpnext/selling/doctype/sales_order/sales_order.js

```diff
@@ -105,6 +105,18 @@ frappe.ui.form.on("Sales Order Item", {
 		} else {
 			frm.script_manager.copy_from_first_row("items", row, ["delivery_date"]);
 		}
+		frappe.call({
+			method: "erpnext.selling.doctype.sales_order.sales_order.get_customer_item_ref_code",
+			args: {	
+				'item': frm.doc.items[0].item_code,
+				'customer_name':frm.doc.customer
```

Comment:
```suggestion
				item: frm.doc.items[0].item_code,
				customer_name: frm.doc.customer
```

Make sure your coding style is consistent.